### ISSUE TITLE
question: what is the status of the `skuid-cli`?

### ISSUE THREAD

**Issue by techfg** [skuid/skuid-cli#236]
TL;DR - What is the status of the `skuid-cli`, is it still actively maintained and supported by Skuid  (Nintex)?  The current release (v0.6.7) is not reliable and in nearly all cases, fails entirely or produces unexpected results.  Can we expect any issues to be addressed/PRs to be merged so that Skuid users can trust and have confidence that the CLI is working as expected in all cases?

The [Skuid Docs](https://docs.skuid.com/latest/en/skuid/cli/) indicate that the `skuid-cli` can and should be used to manage Skuid NLX environments and even encourage its use to aid in managing production environments.  However, except when used under **VERY** specific conditions (for which there are very few of), executing any one of the CLI commands results in:

1. Abnormal termination
2. completion indicating success, but nothing actually changed (either locally or on server) and there is no error/warning
3. completion indicating success, but only partial changes were made (either locally or on server) and there is no error/warning
4. completion indicating error with an inaccurate, meaningless, and/or incredibly difficult to decern error message

Despite these results, the last commit to `master` branch was November 1, 2023 (nearly a year ago).  Since June 24, 2024 (~3.5 months ago), 107 new issues have been opened in this repo, 81 which are bugs (76%) which clearly indicate that there are significant deficiencies with the current version of the CLI.

Of the 107 issues created, only two (2) have been touched by Skuid (on Aug 23, 2024 ~1.5 months ago) and those were minor doc issues, correcting documentation that was already ~16 months out-of-date.  In short, none of the issues in the CLI and/or in the server-side APIs have been addressed nor have the PRs that were submitted, on June 24, 2024 (~3.5 months ago) & Aug 13, 2024 (~2 months ago) to resolve a large portion of the issues been reviewed.

In short, the v0.6.7 should not be used except for under a very specific set of circumstances and **NEVER** in a production environment as the results are not reliable, unpredictable and, if you can even determine that something went wrong, nearly impossible to figure out what caused it unless you spend a good portion of time reviewing the code base.

While it somewhat understandable for a "typical" open-source project to contain bugs/issues (people are maintaining it in their spare time afterall), the `skuid-cli` is not your "typical" open-source project given that it, at least according to the docs, is the official CLI for managing Skuid NLX environments, built, supported and maintained by Skuid directly.  Given it's positioning by Skuid and its commercial backing, there are certain expectations that come with it and being able to use it reliably would be top of the list - something that is not possible currently even with all the fixes in #205.  It is very clear from working on all the issues addressed in #205 that the code itself lacks what would be considered proper test coverage (e.g., there is a tremendous amount of string parsing/manipulation and almost no tests present) and any manual testing that may have been performed was far from adequate.

In summary, I just want to be able to use the CLI that Skuid is telling me I should use to manage my environments and its simply not possible to do so due to all the issues present.  Even with all the fixes in #205, I still can't use it reliably due to server-side API issues.  I've tried everything to make it work "well enough" until everything is fixed but due to issues like #210 & #211, I can't even achieve that outcome.

**Bottom Line**
Is the `skuid-cli` still actively supported and maintained by Skuid (Nintex)?  
- If no, it should be pulled from the Skuid docs immediately, the readme modified to reflect that status and this repo marked archive read-only
- If yes, when can we expect the issues & PRs in this repo to be reviewed/addressed/merged and a new version released so that Skuid users can trust that the CLI is working reliably?  Also, in the meantime, there should be some type of message added to the README (and likely a post to the Skuid forum) about the current limitations/issues so that existing users are aware of the shortcomings.

Looking forward to hearing an official response, thank you!
